src/metrics/metrics.utils.ts

```diff
@@ -30,8 +30,11 @@ export async function getProtocolSettMetrics(): Promise<ProtocolSettsMetrics> {
 }
 
 export async function getChainMetrics(chain: Chain): Promise<ProtocolSummary> {
-  const vaults = await Promise.all(chain.vaults.filter(isStageVault).map((vault) => getCachedVault(chain, vault)));
-  const totalValue = vaults.reduce((total, vault) => total + vault.value, 0);
-  const vaultSummaries = vaults.map(({ name, balance, value }) => ({ name, balance, value }));
+  const vaults = await chain.vaults.all();
+  const chainVaults = await Promise.all(
+    vaults.filter((v) => v.isStageVault()).map((vault) => getCachedVault(chain, vault)),
```

Comment:
What is going on with this `isStageVault()` filter, is this meant to be here? i might misunderstand it's purpose

Reply:
fixed